src/views/Dashboard/GeneticAnalyst/Home/index.vue

```diff
@@ -174,10 +179,12 @@ export default {
 
     async getOrdersData(keyword) {
       this.orderLists = []
+      this.isLoading = true
 
       try {
         let orders = []
         const orderData = await GAGetOrders(keyword)
+        const page = VueRouter?.history?.current
```

Comment:
You can access the router from `this.$router` why use `VueRouter`?

Reply:
updated access from `this.$router`